commit 440a5aff9d52605f8ecf7144925d8cfb173a591c
Author: Kevin Cernekee <cernekee@chromium.org>
Date:   Sat Sep 16 21:08:23 2017 -0700

    brcmfmac: Delete redundant length check
    
    brcmf_fweh_process_event() sets event->datalen to the
    endian-swapped value of event_packet->msg.datalen, which is the
    same as emsg.datalen.  This length is already validated in
    brcmf_fweh_process_event(), so there is no need to check it
    again upon dequeuing the event.
    
    Suggested-by: Arend van Spriel <arend.vanspriel@broadcom.com>
    Signed-off-by: Kevin Cernekee <cernekee@chromium.org>

diff --git a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/fweh.c b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/fweh.c
index ef72baf6dd96..e7eaa57d11d9 100644
--- a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/fweh.c
+++ b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/fweh.c
@@ -257,11 +257,6 @@ static void brcmf_fweh_event_worker(struct work_struct *work)
 		brcmf_dbg_hex_dump(BRCMF_EVENT_ON(), event->data,
 				   min_t(u32, emsg.datalen, 64),
 				   "event payload, len=%d\n", emsg.datalen);
-		if (emsg.datalen > event->datalen) {
-			brcmf_err("event invalid length header=%d, msg=%d\n",
-				  event->datalen, emsg.datalen);
-			goto event_free;
-		}
 
 		/* special handling of interface event */
 		if (event->code == BRCMF_E_IF) {
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1143_440a5aff9d52605f8ecf7144925d8cfb173a591c.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1143_440a5aff9d52605f8ecf7144925d8cfb173a591c.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
